pilot/pkg/bootstrap/server.go

```diff
@@ -273,6 +277,23 @@ func NewServer(args *PilotArgs) (*Server, error) {
 		return nil, fmt.Errorf("config validation: %v", err)
 	}
 
+	if dns.DNSAddr.Get() != "" {
+		if err := s.initDNSTLSListener(dns.DNSAddr.Get()); err != nil {
+			// TODO: will be made an error before launching. Tests are currently
```

Comment:
sounds like we should not merge without this?

Reply:
Removed the TODO - there are few other things we don't crash for, and there are other ways to 
deal this ( probers, readiness, etc).

Reply:
Also it's expected that in 1.6 only select security-sensitive or MC workloads will use this feature, so Fatalf Istiod for everyone else is not a good idea.